variables.tf

```diff
@@ -619,6 +620,7 @@ variable "runner_worker_docker_autoscaler" {
     max_use_count                  = optional(number, 100)
     update_interval                = optional(string, "1m")
     update_interval_when_expecting = optional(string, "2s")
+    instance_ready_command         = optional(string)
```

Comment:
Based on the [comment](https://gitlab.com/gitlab-org/gitlab/-/issues/408131#note_2099525433), we could potentially set the default to "cloud-init status --wait" if preferred

Reply:
or "cloud-init status --wait || test $? -eq 2" according to [discussion](https://gitlab.com/hetznercloud/fleeting-plugin-hetzner/-/issues/3#note_1900817924) about new cloud init version

Reply:
As I am not sure what the best default is, will leave this empty.

Reply:
I think, we should use an empty default value. In case you use your own AMI the `cloud-init` might not always be available.

Reply:
Thanks for comment. 
Did some more testing with the empty "" value.  The empyt value seems to work fine for spinning up runners but only if we make sure the value is wrapped to be a string on the config.toml.